deployment/Dockerfile-frontend

```diff
@@ -0,0 +1,64 @@
+# To build a Docker image from this file, run from the root directory:
+# docker build -f deployment/Dockerfile-frontend -t hansel-frontend .
+
+# Intermediate base image
+FROM node:16.9.1 AS vue-build
+
+# Install Protocol Buffer compiler
+RUN apt-get update
+RUN apt-get install -y protobuf-compiler
+
+# Environment variables
+ENV API_SPECS_ROOT /api-specs
+ENV FRONTEND_ROOT /frontend
+
+# Copy API specs into container
+RUN mkdir --parents $API_SPECS_ROOT
+WORKDIR $API_SPECS_ROOT
+COPY api-specs .
+
+# Copy source code into container
+RUN mkdir --parents $FRONTEND_ROOT
+WORKDIR $FRONTEND_ROOT
+COPY frontend .
+
+# Install dependencies
+RUN npm ci
+
+# Build app
+RUN npm run build
+
+# Base image
+FROM nginx:1.21.3
+
+# Environment variables
+ENV APP_ROOT /build
+
+# Make app directory
+RUN mkdir --parents $APP_ROOT
+WORKDIR $APP_ROOT
+
+# Copy Vue build
+COPY --from=vue-build /frontend/dist .
+
+# Delete NGINX defaults (does not work if removed)
```

Comment:
do you mean 
```suggestion
# Delete NGINX defaults (does not work if not removed)
```
?

Reply:
I actually meant the line itself in the Dockerfile, but now that you mention it, I think your interpretation makes more sense.